### PR TITLE
Set up coveralls

### PR DESCRIPTION
**Pull request by abhinav**

### CODE REVIEW

**Comment by abhinav** [comment]
@blampe @breerly

**Comment by breerly** [comment]
:sheep:

File: Makefile [hunk 1]
```diff
@@ -24,4 +23,25 @@ install:
 
 .PHONY: test
 test:
-	go test $(PACKAGES) $(TEST_ARGS)
+	go test $(PACKAGES) -v
+
+.PHONY: cover
+cover:
```
Comment (prashantv) [inline Makefile]: do we need cover if we don't use it to generate the coverage files?

File: Makefile [hunk 2]
```diff
@@ -24,4 +23,25 @@ install:
 
 .PHONY: test
 test:
-	go test $(PACKAGES) $(TEST_ARGS)
+	go test $(PACKAGES) -v
```
Comment (prashantv) [inline Makefile]: I think `-race` got dropped here. does `goveralls` run with `-race`?

Comment (abhinav) [inline Makefile]: goveralls doesn't support `-race`

File: Makefile (hunk 1 above)
Comment (abhinav) [inline Makefile]: It still mentions the percentages on the command line.

**Pull request merged**